test/e2e/push_test.go

```diff
@@ -105,7 +106,9 @@ var _ = Describe("Podman push", func() {
 		}
 		lock := GetPortLock("5000")
 		defer lock.Unlock()
-		session := podmanTest.Podman([]string{"run", "-d", "--name", "registry", "-p", "5000:5000", REGISTRY_IMAGE, "/entrypoint.sh", "/etc/docker/registry/config.yml"})
+		// FIXME. This uses registry-1.docker.io to bypass a registries.conf mirror configuration
+		// Actually we need to mirror that image.
+		session := podmanTest.Podman([]string{"run", "-d", "--name", "registry", "-p", "5000:5000", "registry-1.docker.io/library/registry:2.8.1", "/entrypoint.sh", "/etc/docker/registry/config.yml"})
```

Comment:
The current `REGISTRY_IMAGE` is `quay.io/libpod/registry:2.6`, which does not support OCI artifacts.

So the `quay.io/libpod/registry` image should be updated, to at least 2.7. And there already seems to be a 2.8 image in that repo – so let me try updating that, in a separate PR.

Reply:
#15122